contracts/lib/LibBalances.sol

```diff
@@ -103,6 +103,7 @@ library Balances {
 
         uint256 notionalValue = netValue(position, price);
 
+        // todo confirm that liquidation gas cost should be a WAD value
         uint256 liquidationGasCost = liquidationCost * 6;
 
         uint256 minimumMarginWithoutGasCost = notionalValue / maximumLeverage;
```

Comment:
I'm wondering if this should be PRBMath.div? maximumLeverage, I believe, is a WAD value, as is notionalValue.

Reply:
@BenjaminPatch good catch I think you are right on maximumLeverage being a WAD value. Will check and update